test-suite/coq-makefile/timing/run.sh

```diff
@@ -40,7 +40,7 @@ INFINITY_REPLACEMENT="+.%" # assume that if the before time is zero, we expected
 
 TO_SED_IN_BOTH=(
     -e s"/${INFINITY}/${INFINITY_REPLACEMENT}/g" # Whether or not something shows up as ∞ depends on whether a time registers as 0.s or as 0.001s, so we can't rely on this being consistent
-    -e s":|\s*N/A\s*$:| ${INFINITY_REPLACEMENT}:g" # Whether or not something shows up as N/A depends on whether a time registers as 0.s or as 0.001s, so we can't rely on this being consistent
+    -e s":|\s*N/A\s*\$:| ${INFINITY_REPLACEMENT}:g" # Whether or not something shows up as N/A depends on whether a time registers as 0.s or as 0.001s, so we can't rely on this being consistent
```

Comment:
It might be better to alternate quote types here, ie
```bash
':|\s*N/A\s*$:| '"$INFINITY_REPLACEMENT"':g'
```
Pick according to your aesthetic sensibilities.

Reply:
Updated